src/plugins/dbusrecv/README.md

```diff
@@ -0,0 +1,45 @@
+- infos = Information about the dbus plugin is in keys below
+- infos/author = Markus Raab <elektra@libelektra.org>
+- infos/licence = BSD
+- infos/provides = notification
```

Comment:
maybe we should give a different provider name for your async io notification receiving and sending plugins?

Reply:
I also call them "transport plugins". Maybe `notificationtransport`?